File: pinyougou-manager-web/src/main/webapp/js/pinyougou/service/sellerService.js
app.service("sellerService",function($http){

    this.findAll=function(){
        return $http.get("../seller/findAll.do");
    }

    this.findPage=function(page,rows,seller){
        return $http.post("../seller/findPage.do?page="+page+"&rows="+rows,seller);
    };

    this.save=function(seller){
        return $http.post("../seller/save.do",seller);
    };

    this.findOne=function(id){
        return $http.get("../seller/findOne.do?id="+id);
    };

    this.deleteBatch=function(ids){
        return $http.get("../seller/delete.do?ids="+ids);
    }

    this.updateStatus=function (sellerId, status) {
        return $http.get("../seller/updateStatus.do?sellerId="+sellerId+"&status="+status);
    }
});

File: pinyougou-sellergoods-interface/src/main/java/com/rogercw/pinyougou/service/SellerService.java
package com.rogercw.pinyougou.service;
import java.util.List;
import com.rogercw.pinyougou.pojo.Seller;

import com.rogercw.page.PageResult;
/**
 * @Author: rogercw
 * @Version 1.0
 */
public interface SellerService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<Seller> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(Seller seller);
	
	
	/**
	 * 修改
	 */
	public void update(Seller seller);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public Seller findOne(String id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(String[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(Seller seller, int pageNum, int pageSize);


	/**
	 * 更新商家状态
	 * @param status
	 */
	void updateStatus(String sellerId,String status);
}
